public/js/utils/redux/middleware/log.js

```diff
@@ -9,7 +9,7 @@
  */
 function log({ dispatch, getState }) {
   return next => action => {
-    console.log("[DISPATCH]", JSON.stringify(action, null, 2));
+    console.log(`[DISPATCH ${action.type}]`, action);
```

Comment:
We had it like this before, but changed it to print the entire object because it got way too annoying to always expand the properties to inspect stuff. Usually you are scanning through many actions to see what happened, so we ended up having to manually expand every single one. For the type of debugging this allows, reading the action log is more important than interacting with it...

How do use action logging? Is there something about printing the object that you don't like?